src/core_plugins/kibana/public/dashboard/listing/dashboard_landing_page_table.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import { LandingPageTable } from 'ui_framework/components';
+import { getDashboardColumns } from './get_dashboard_columns';
+import { ItemTableActions } from 'ui/saved_object_table/item_table_actions';
+import { DashboardConstants } from '../dashboard_constants';
+
+export function DashboardLandingPageTable({ itemTableState, onDeleteItems, onFilterItems, isFetchingItems }) {
```

Comment:
Is "connecting logic to components" not exactly what it does already by partially applying actions and adapting the state to whatever format the inner components required? There are some issues with the way it does that right now, but in general I would argue against "filling it with more logic". If this encapsulated the `<ControlledTable>...</ControlledTable>` (which looks like a good idea to me) it would probably appear less "superfluous".